tests/auto/gltfexporter/tst_gltfexporter.cpp

```diff
@@ -33,43 +33,112 @@
 #include <Kuesa/GLTF2Exporter>
 #include <Kuesa/private/gltf2parser_p.h>
 #include <Kuesa/private/kuesa_utils_p.h>
-
 using namespace Kuesa;
 using namespace GLTF2Import;
 
 class tst_GLTFExporter : public QObject
 {
     Q_OBJECT
 
+    QDir setupTestFolder()
```

Comment:
If would even go for a free standing function in an anonymous namespace

Reply:
done